realsense_ros/CMakeLists.txt

```diff
@@ -100,6 +101,7 @@ target_link_libraries(${PROJECT_NAME}
 )
 
 ament_target_dependencies(${PROJECT_NAME}
+  rcl
```

Comment:
As only the include is used, maybe `target_include_directories` would be more accurate. Not sure what `ament_target_dependencies` does to the target but likely more things that we one would need/want here.

Reply:
I added e47e1d5 , but I kind of like leaving it to just `ament_target_dependencies`. Anything rclcpp related needs rcl any how.